scripts/basic-workflow-tests.sh

```diff
@@ -145,15 +145,15 @@ test_download() {
     fi
 
     if [ "$MIXTRAL" -eq 1 ]; then
-        step Downloading the mixtral model as the teacher
+        step Downloading the mixtral model as the teacher model for SDG
         ilab model download --repository TheBloke/Mixtral-8x7B-Instruct-v0.1-GGUF --filename ${MIXTRAL_GGUF_MODEL} --hf-token "${HF_TOKEN}"
     else
-        step Downloading the default merlinite .gguf model as the teacher
+        step Downloading the default merlinite .gguf model as the teacher model for SDG
         ilab model download --repository instructlab/merlinite-7b-lab-GGUF --filename ${MERLINITE_GGUF_MODEL}
     fi
 
     if [ "$EVAL" -eq 1 ]; then
-        step Downloading merlinite .safetensors model to use in evaluation as the judge model
+        step Downloading merlinite .safetensors model to use in Evaluation as the judge model
```

Comment:
Is this to be consistent?

Reply:
Yup - everywhere else I see capitalization being used

Reply:
I am not seeing the other examples I guess.  Other than the acronym of SDG.

Reply:
I was doing it based on:
- https://github.com/instructlab/instructlab/blob/main/scripts/basic-workflow-tests.sh#L367
- https://github.com/instructlab/instructlab/blob/main/scripts/basic-workflow-tests.sh#L469

Reply:
Talked on Slack